### ISSUE TITLE
bug(fileUploader): has DAP violations 

### ISSUE THREAD

**Issue by abbeyhrt** [carbon-design-system/carbon#4208]
The fileUploader has a DAP violation

Here is the DAP violation: 
<img width="714" alt="Screen Shot 2019-10-03 at 3 07 15 PM" src="https://user-images.githubusercontent.com/17281178/66160568-07f63b00-e5f0-11e9-9d02-2b07c88c5731.png">

**Comment by emyarod** [comment]
this attribute is carried over from the existing component, so do we want to remove the `role` attribute from the legacy file uploader?

it seems to originate from https://github.com/carbon-design-system/carbon/commit/6446ea5603c79896a920cdf71bdde6cb057e5efd

related #3316 #3362

**Comment by abbeyhrt** [comment]
I'm going to close this issue since a duplicate was made. #4221 will cover this problem now!
